### PR TITLE
Added some notes about Japanese Support

### PR DESCRIPTION
**Pull request by alexandruatomei9**

### CODE REVIEW

File: salesforce/10.4/modules/ROOT/pages/salesforce-connector-troubleshoot.adoc [hunk 1]
```diff
@@ -42,9 +42,18 @@ You must know the scope of the rate limiting policy that applies to your account
 
 When extracting data from a Salesforce Opportunity object, be aware that a "quarter" in this context is relative to the financial year of the organization and not necessarily to the calendar year.
 
+== Japanese Support
+
+Salesforce Connector supports Japanese characters for most operations, with these exceptions:
+
+* The *Publish Streaming Channel* operation does not support Japanese characters in the *Streaming Channel Name*, which can contain alphanumeric and `&#95;`, `/` characters only. See https://developer.salesforce.com/docs/atlas.en-us.228.0.object_reference.meta/object_reference/sforce_api_objects_streamingchannel.htm[Salesforce Streaming channel documentation].
```
Comment (dhirsch1) [inline salesforce/10.4/modules/ROOT/pages/salesforce-connector-troubleshoot.adoc]: "Streaming Channel Name field" (with the asterisks around the field name, like you have)

This is not clear: "...which can contain alphanumeric and _, / characters only" Do you mean to say this? "...which can contain alphanumeric characters and the `_ ` or `/` characters only"?

File: salesforce/10.4/modules/ROOT/pages/salesforce-connector-troubleshoot.adoc [hunk 2]
```diff
@@ -42,9 +42,18 @@ You must know the scope of the rate limiting policy that applies to your account
 
 When extracting data from a Salesforce Opportunity object, be aware that a "quarter" in this context is relative to the financial year of the organization and not necessarily to the calendar year.
 
+== Japanese Support
+
+Salesforce Connector supports Japanese characters for most operations, with these exceptions:
```
Comment (dhirsch1) [inline salesforce/10.4/modules/ROOT/pages/salesforce-connector-troubleshoot.adoc]: Suggestion: 
"Salesforce Connector supports Japanese characters for all operations, with these exceptions:"
or
"Salesforce Connector supports Japanese characters for all operations, except for the following:"

File: salesforce/10.4/modules/ROOT/pages/salesforce-connector-troubleshoot.adoc [hunk 3]
```diff
@@ -42,9 +42,18 @@ You must know the scope of the rate limiting policy that applies to your account
...
+
+* The *Publish Streaming Channel* operation does not support Japanese characters in the *Streaming Channel Name*, which can contain alphanumeric and `&#95;`, `/` characters only. See https://developer.salesforce.com/docs/atlas.en-us.228.0.object_reference.meta/object_reference/sforce_api_objects_streamingchannel.htm[Salesforce Streaming channel documentation].
+
+* Metadata operations might return error messages if some fields contain Japanese characters. For example, creating custom objects or custom fields with a fullName that contains Japanese characters triggers a `FIELD_INTEGRITY_EXCEPTION` with the following message: +
```
Comment (dhirsch1) [inline salesforce/10.4/modules/ROOT/pages/salesforce-connector-troubleshoot.adoc]: fullName > full name?

Comment (valkyrie69) [reply salesforce/10.4/modules/ROOT/pages/salesforce-connector-troubleshoot.adoc]: That's a parameter--it's fullName.

File: salesforce/10.4/modules/ROOT/pages/salesforce-connector-troubleshoot.adoc (hunk 2 above)
Comment (valkyrie69) [reply salesforce/10.4/modules/ROOT/pages/salesforce-connector-troubleshoot.adoc]: I don't think I should change it to all. They may not have tested all operations to make sure that's the case. Alex stated most in his original wording so I think it's safer to keep that.

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**